src/test/java/api/support/fixtures/CirculationRulesFixture.java

```diff
@@ -66,12 +69,20 @@ public void updateCirculationRules(String rules) {
       .put("rulesAsText", rules);
 
     final Response response = putRules(circulationRulesRequest.encodePrettily());
-
     assertThat(String.format(
       "Failed to set circulation rules: %s", response.getBody()),
       response.getStatusCode(), is(204));
   }
 
+  public void attemptUpdateCirculationRules(String rules, String policyType) {
```

Comment:
Attempt methods usually do not make the assertions themselves, they are intended for use in tests that expect specific failure conditions to occur.

If this method is intended to demonstrate a specific failure, I suggest renaming it to reflect that situation. If it is only used in a single place, maybe it could be inlined.

Reply:
OK, renamed